app/frontend/courses/review/components/CoursesReview__Editor.res

```diff
@@ -982,7 +951,7 @@ let feedbackGenerator = (
               value=state.newFeedback
               profile=Markdown.Permissive
               maxLength=10000
-              disabled={isSubmissionReviewAllowed(submissionDetails)}
+              disabled={!isSubmissionReviewAllowed(submissionDetails)}
```

Comment:
why not create a new function like `isReviewDisabeld` 🤔 ; Make the code more readable;

Reply:
Every usage of this function does seem to need a negation in front of it. That's usually a red flag on naming. 😅